### ISSUE TITLE
Release?

### ISSUE THREAD

**Issue by rsignell-usgs** [informatics-lab/tiledb_netcdf#32]
Seems it would be nice to have a release, now that the [blog post](https://medium.com/informatics-lab/storing-cloud-ready-geoscience-data-with-tiledb-34d454c33055) is out?

**Comment by DPeterK** [comment]
Yeah, getting this released is reasonable! Here's what I think needs to happen first:

v0.1 (easy)
* [ ] fix #34 
* [ ] simplify the `TDBWriter` subclasses, or at least present only one public API for them
* [ ] Update the README to represent the current state of the public API

v1.0 (desirable)
* [ ] fix #33 and address the underlying question of how to deal with undetected data variables
* [ ] reduce duplication in `writers.py` - perhaps going as far as removing the single attribute writer class entirely
* [ ] lots more documentation (and upload it to readthedocs?)
* [ ] tests! any tests, lots of tests
* [ ] CI
* [ ] publishing the codebase
